src/services/api/realm-configuration.js

```diff
@@ -0,0 +1,28 @@
+
+import ApiBase, { tryCastToApiError, toUrlParams, isValid, } from './api-base';
+
+export default class RealmConfigurationAPI extends ApiBase
+{
+  constructor(options) {
+    super(options);
+  }
+
+  async queryRealmConfigurations({q, f, s, sk, l , c, fo, ag }={})  { 
+    const params = toUrlParams( {q, f, s, sk, l , c, fo, ag });
+
+    return this.http.get(`/api/v2018/realm-configurations`,  { params } )
+                    .then(res => res.data)
+                    .catch(tryCastToApiError);
+  }
+
+  async getRealmConfigurationByHost(host)  {
```

Comment:
if `host` is optional maybe it should be passed using `{ host }`
```suggestion
  async getRealmConfigurationByHost({ host }={})  {
```